themes/bootstrap5/scss/bootstrap-variable-overrides.scss

```diff
@@ -21,7 +21,6 @@ $danger-border-subtle:      darken(adjust-hue($danger-bg-subtle, -10), 5%) !defa
 $input-border-color:        #888 !default;
 $form-select-border-color:  #888 !default;
 
-$breadcrumb-bg:                     #f5f5f5 !default;
```

Comment:
As I understand it, we would have to not define any colors using these bootstrap global properties, as these are not specific to any color mode.